modules/light-clients/06-solomachine/header.go

```diff
@@ -10,6 +10,9 @@ import (
 	"github.com/cosmos/ibc-go/v6/modules/core/exported"
 )
 
+// SentinelHeaderPath defines a placeholder path value used for headers in solomachine client updates
+const SentinelHeaderPath = "solomachine:header"
```

Comment:
should we match the proposed spec value? https://github.com/cosmos/ibc/pull/821/files#diff-028a9db43f8be864645baa5997fbb182b9af4cd9d444166944920c9fb25ee5fcR179

I'm okay proposing a different value in the spec as well

cc @AdityaSripal

Reply:
Huh! I missed this was already part of the spec, my bad. 
I actually feel like proposing a change, what do you think? 

In my opinion `solomachine:header` feels a little easier on the eye than the capitalised snake case. Don't feel too strongly about it but just my two cents

Reply:
Sounds good to me!

Reply:
https://github.com/cosmos/ibc/pull/821/files#r1025165561